### DISCUSSION TITLE
Help!

### DISCUSSION THREAD

**Discussion by Independiente-work** [community/community#53499] [Programming Help] [answered]
### Select Topic Area

Question

### Select Feature Area

- [X] Issues
- [X] Projects

### Body

https://github.com/Independiente-work/cryptosocial/issues/1#issue-1679639456

This is my first code for which I want to build a logic to output a list of 50 coins with a determined scoring. The 4 errors shown dont seem to go away :(

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @Independiente-work ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
